packages/driver/src/cy/commands/actions/trigger.js

```diff
@@ -5,8 +5,29 @@ const $dom = require('../../../dom')
 const $errUtils = require('../../../cypress/error_utils')
 const $actionability = require('../../actionability')
 
-const dispatch = (target, eventName, options) => {
-  const event = new Event(eventName, options)
+const dispatch = (target, window, eventName, options) => {
```

Comment:
I prefer that we use `win` instead of `window` in cases like this, so it doesn't get confused with the actual `window` global for the current frame.

Reply:
Or, on second thought, maybe it should be `appWindow` to make it absolutely clear what window it is.

Reply:
`win` is used a lot for AUT window and `appWindow` isn't used at all in the code base. But I also think `appWindow` is much clearer than `win`. 

Maybe, it should start using it from now on.

Reply:
I agree about using `appWindow` from now on even though we haven't used it before. Between the top window, spec window, and app window, it can get confusing which one is being referenced.